.github/workflows/push-gradle-ci.yml

```diff
@@ -39,6 +39,36 @@ jobs:
     - name: Build with Gradle Wrapper
       run: ./gradlew release
 
+    - name: Upload release
+      if: github.repository == 'ion-fusion/fusion-java'
+      env:
+        GH_TOKEN: ${{ github.token }}
+      run: |
+        # Define release tag and name.
+        RELEASE_TAG="latest"
+        RELEASE_TITLE="Latest build"
```

Comment:
At the moment, let's stick with "latest" since that's the (default) term used by Read The Docs for the build from tip of `main`.

I'm not tied to it, but those should be aligned.

Reply:
By default, ReadTheDocs uses the term "stable" for the most recent versioned release.